### ISSUE TITLE
Can you reupload pretrained model?

### ISSUE THREAD

**Issue by 0312birdzhang** [spacewalk01/yolov5-fire-detection#12]
[yolov5s_best.pt](https://github.com/spacewalk01/yolov5-fire-detection/blob/main/models/yolov5s_best.pt) is only 2bit, seems like it's not a model.

**Comment by spacewalk01** [comment]
I see. It seems like there something went wrong, when I renamed the model file. I will check it. Thanks.

**Comment by spacewalk01** [comment]
Here is the correct file
https://github.com/spacewalk01/yolov5-fire-detection/blob/5f9317e1f6deb08ac07fa04deca22569a7318aef/models/best.pt

The current model file seems crashed.

**Comment by spacewalk01** [comment]
Re-uploaded the model file

**Comment by trohit920** [comment]
Hey @spacewalk01,
Thanks for this work but I have a question related to accuracy of the model you trained.
Can you share the map50 and map50-95 value you achieved on validation dataset. 

Here is my result, please let me do if i am doing something wrong?
```
val: data=../fire.yaml, weights=['../model/yolov5s_best.pt'], batch_size=8, imgsz=640, conf_thres=0.001, iou_thres=0.5, max_det=300, task=val, device=, workers=8, single_cls=False, augment=False, verbose=False, save_txt=False, save_hybrid=False, save_conf=False, save_json=True, project=../../yolov5/runs/val, name=exp, exist_ok=False, half=False, dnn=False
YOLOv5 🚀 v7.0-304-g22361691 Python-3.10.12 torch-2.2.1+cu121 CUDA:0 (Tesla T4, 15102MiB)

Fusing layers... 
Model summary: 213 layers, 7012822 parameters, 0 gradients, 15.8 GFLOPs
Downloading https://ultralytics.com/assets/Arial.ttf to /root/.config/Ultralytics/Arial.ttf...
100% 755k/755k [00:00<00:00, 25.1MB/s]
/usr/lib/python3.10/multiprocessing/popen_fork.py:66: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.
  self.pid = os.fork()
val: Scanning /content/yolov5-fire-detection/datasets/val/labels... 150 images, 0 backgrounds, 0 corrupt: 100% 150/150 [00:00<00:00, 907.23it/s]
val: New cache created: /content/yolov5-fire-detection/datasets/val/labels.cache
                 Class     Images  Instances          P          R      mAP50   mAP50-95: 100% 19/19 [00:07<00:00,  2.46it/s]
                   all        150        256      0.564       0.41      0.448      0.184
Speed: 0.2ms pre-process, 12.0ms inference, 7.5ms NMS per image at shape (8, 3, 640, 640)

Evaluating pycocotools mAP... saving ../../yolov5/runs/val/exp2/yolov5s_best_predictions.json...
loading annotations into memory...
Results saved to ../../yolov5/runs/val/exp2
```

**Comment by spacewalk01** [comment]
Hi, it seems right. I remember I got the similar results after 5 epochs. You need to train for more epochs to get better results.

**Comment by trohit920** [comment]
@spacewalk01 thanks for your prompt response. Do you think I can get the better result with Yolov8? My target is to get 95% map50 on validation data so planning to train from scratch.

**Comment by spacewalk01** [comment]
I think so you can get better results with yolov8 or v9.
